index.d.ts

```diff
@@ -0,0 +1,151 @@
+
+declare namespace aperture {
+  type Screen = {
+    id: number;
+    name: string;
+  }
+
+  type AudioDevice = {
+    id: string;
+    name: string;
+  }
+
+  type VideoCodec = 'h264' | 'hevc' | 'proRes422' | 'proRes4444'
+
+  type RecordingOptions = {
+    /**
+     * Number of frames per seconds.
+     */
```

Comment:
```suggestion
    /**
    Number of frames per seconds.
    */
```

Drop the star vertical border. Makes it too hard to edit the comments. Applies to this whole file.

Reply:
Makes sense. I wonder if there's an editor setting to disable it. I'll make the updates today and push